src/sources/available_screen_position.ts

```diff
@@ -0,0 +1,39 @@
+import { replaceNaN, round, toFloat } from '../utils/data'
+import * as browser from '../utils/browser'
+
+type Position = [left: number | null, top: number | null, width: number | null, height: number | null]
+
+const roundingPrecision = 10
+
+export default function getAvailableScreenPosition(): Position | undefined {
+  if (!isUseful()) {
+    return undefined
+  }
+
+  // Some browsers return screen resolution as strings, e.g. "1200", instead of a number, e.g. 1200.
+  // I suspect it's done by certain plugins that randomize browser properties to prevent fingerprinting.
+  //
+  // Sometimes the resolution changes a bit for unknown reason, e.g. 1900x1440 → 1900x1439,
+  // the rounding is used to mitigate this difference.
+  const prepareValue = (value: unknown) => replaceNaN(round(toFloat(value), roundingPrecision), null)
```

Comment:
just FYI, you can also use `+value` for string to integer conversion, which is faster